src/components/ActivitiesList/ActivityList.tsx

```diff
@@ -0,0 +1,24 @@
+import ListView from "../ItemList/ItemList";
+import { Activity } from "./Definition";
+
+interface ActivityListProps {
+  activities: Activity[];
+  className?: string;
```

Comment:
```suggestion
```
This is a new pattern for us, where we pass one or more classnames into a reusable component, so we can send all the styling from the "consumer" (the component that renders the reusable component)

So we want to leave the `className` prop on `ItemList`, but not on the different lists that consume it:

```
`ActivitiesList.module.css`
  * define `.example` class styling
`ActivitiesList.tsx`
  * import `styles` from module
  * pass `styles.example` to `ItemList` `className`
`ItemList.tsx`
  * receive `className` prop
  * apply `className` to element that needs styling
```

the benefit of doing things this way is that theoretically we'd create css modules for every different consumer of the `ItemList`, eg  `ActivitiesList`, `VotesList`, `GroupsList`, `VotingList`, `MemberList`, `EventList` etc

Then every one of those components passes a classname to the `ItemList` which means that we can style the list 10 different ways for 10 different consumers, without having to have any of that styling or logic live inside of the generic list

We can look at some more examples of this later, but in the meantime you can remove this line and then add some test styling if you want via `ActivitiesList.module.css`

Reply:
Fixed! [4cb382a
](https://github.com/Code-the-Dream-School/ee-prac-team2-front/pull/21/commits/4cb382a2718c3ee68ce03b1834ecd8f181c3b202)